examples/userInterface/userInteface.html

```diff
@@ -0,0 +1,164 @@
+<html>
+
+    <head>
+        <title>OpenGlobus - Earth planet</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="../../css/og.css" type="text/css" />
+        <style>
+            body {
+                margin: 0px;
+                padding: 0px;
+            }
+
+        </style>
+    </head>
+
+    <body>
+        <div id="globus" style="width:100%;height:80%"></div>
+
+
+        <script type="module">
+            'use strict';
+
+            import { Globe } from '../../src/og/Globe.js';
+            import { XYZ } from '../../src/og/layer/XYZ.js';
+            import { GeoImage } from '../../src/og/layer/GeoImage.js';
+            import { GeoImageDragControl } from '../../src/og/control/GeoImageDragControl.js';
+            import { LayerSwitcher } from '../../src/og/control/LayerSwitcher.js';
+            import { GlobusTerrain } from '../../src/og/terrain/GlobusTerrain.js';
+            import { Ruler } from "../../src/og/control/ruler/Ruler.js";
+            import { Vec3 } from '../../src/og/math/Vec3.js';
+            import { LonLat } from '../../src/og/LonLat.js';
+            import { Entity } from '../../src/og/Entity/Entity.js';
+            import { wgs84 } from '../../src/og/ellipsoid/wgs84.js';
+            import { Vector } from '../../src/og/layer/Vector.js';
+            import { RADIANS } from '../../src/og/math.js';
+
+            let osm = new XYZ("OpenStreetMap", {
+                isBaseLayer: true,
+                url: "//{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",
+                visibility: true,
+                attribution: 'Data @ OpenStreetMap contributors, ODbL',
+
+            });
+
+            var sat1 = new XYZ("MapQuest Satellite", {
+                fading: true,
+                shininess: 20,
+                specular: [0.00048, 0.00037, 0.00035],
+                diffuse: [0.88, 0.85, 0.8],
+                ambient: [0.15, 0.1, 0.23],
+                isBaseLayer: true,
+                url: "https://api.mapbox.com/styles/v1/mapbox/satellite-v9/tiles/256/{z}/{x}/{y}?access_token=pk.eyJ1IjoibWdldmxpY2hzY2FuZXgiLCJhIjoiY2pwcGdsaXlnMDQzdDQybXhsOWZlbXBvdSJ9.fR2YE-ehJA4iajaJBAPKvw",
+                visibility: false,
+                attribution: `@2014 MapQuest - Portions @2014 "Map data @
+            <a target="_blank" href="//www.openstreetmap.org/">OpenStreetMap</a> and contributors,
+            <a target="_blank" href="//opendatacommons.org/licenses/odbl/"> CC-BY-SA</a>"`
+            });
+
+            var sat2 = new XYZ("MapQuest Sat Labels", {
+                fading: true,
+                shininess: 20,
```

Comment:
Btw, shininess, specular, diffuse, and ambient parameters do not exist anymore. It was a long time ago.

Reply:
Done.